### DISCUSSION TITLE
Discord Request

### DISCUSSION THREAD

**Discussion by Goldenfreddy0703** [Goldenfreddy0703/plugin.video.kaito.beta#46] [Q&A]
Hey @Gujal00 , sorry to bother you but i didn't know where to ask you except here, Would it be alright to have you on Discord? We cam discuss alot and where the direction this addon is going. 

Here is my Discord: The Steampunk Owl#3126

**Comment by Gujal00** [comment]
I do not have discord account. I avoid all social media accounts
The best place to reach me https://keybase.io/gujal which is an opensource encrypted chat. I use it on my desktop and android mobile and the ResolveURL discussion group is also there

**Reply by Goldenfreddy0703** [reply]
Ahh i understand but if you do decide to make a discord account, you should join me, me and a couple of kodi developers have a community called Addons4kodi and they provide the best help especially with there skins. I will give keybase i try.

**Reply by Gujal00** [reply]
Keybase has no voice support. Only chat and attachments. you get 250GB encrypted storage for free as well

**Reply by Goldenfreddy0703** [reply]
Oh then that's fine

**Reply by Goldenfreddy0703** [reply]
Hey i just made an account and verified and messaged you on there. Did you receive it?
